app/assets/translations/appConstants/bebbo/RSen.ts

```diff
@@ -392,10 +389,15 @@ const RSen = {
                 "parentGender": "Gender",
                 "successOnboardingImport": "You have successfully uploaded your  information.",
                 "updateImportText": "Please update your details to continue",
-                "importOnboardingText": "You can upload a backed up data file",
+                "importOnboardingText":"Bebbo does not collect your personal details.",
+                "importOnboardingText1": "Information you enter to the app is saved to your phone or your personal storage.",
```

Comment:
I would write as "The information you input into the app is stored on your phone or personal storage."
but it is fine for now.